README.md

```diff
@@ -52,6 +52,22 @@ application should be configured to be a **Regular Web Application**.
 * `AUTH0_DOMAIN`: The Auth0 application's domain
 * `AUTH0_KEY`: The Auth0 application's client id
 * `AUTH0_SECRET`: The Auth0 application's client secret.
+* `TWILIO_ACCOUNT_SID`: Twilio Account SID for client
+* `TWILIO_AUTH_TOKEN`: Twilio Auth Token for client
+* `TWILIO_SMS_NUMBER`: Phone number received from Twilio for client
+
+## Running NGROK
+NGROK is needed to manually test sending text messages
+
+- In your root directory:
```

Comment:
while both '-' and '\*' will both create unordered lists, given that the rest of the README uses '*', it would be good to be consistent for readability. I don't personally care which one to use, so if you want to use '-', could you change all the other lists to it?